libs/api/domains/company-registry/src/lib/resolvers/rskCompanyInfo.resolver.ts

```diff
@@ -0,0 +1,36 @@
+import { UseGuards } from '@nestjs/common'
+import { Parent, ResolveField, Resolver } from '@nestjs/graphql'
+
+import {
+  CurrentUser,
+  IdsUserGuard,
+  Scopes,
+  ScopesGuard,
+} from '@island.is/auth-nest-tools'
+import type { User } from '@island.is/auth-nest-tools'
+import { ApiScope } from '@island.is/auth/scopes'
+
+import { RskCompanyRelatedParty } from '../models/rskCompanyRelatedParty.model'
+
+import { RskCompany, RskCompanyInfo } from '../models/rskCompany.model'
+import { RskCompanyInfoService } from '../rsk-company-info.service'
+
+@UseGuards(IdsUserGuard, ScopesGuard)
+@Scopes(ApiScope.internal, ApiScope.company, ApiScope.serviceDesk)
```

Comment:
We need to test and make sure that this field resolver only works if you have the serviceDesk scope.